src/index.js

```diff
@@ -14,13 +14,13 @@ module.exports = class Layer {
 
     var child = this._render()
 
-    return React.render(child, this._mountPoint, cb);
+    return ReactDOM.render(child, this._mountPoint, cb);
```

Comment:
oof yeah I forgot about that...yeah that's a good option, make it optional. Although prehaps its better to make it an argument on `layer.render()` instead of the constructor?

I'd appreciate it if you could update the docs as well on that point :) thanks!